src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -154,17 +112,24 @@ public void setUrl(URL url) {
         // System.out.println(_url.toString());
     }
 
-    public void setKey(String key) {
+    public void setKey() throws IOException { // try to find key file and set API key

```

Comment:
Same here - could you move the current in-line `//` comment to Javadocs on the method itself?

Reply:
BTW - In IntelliJ you can auto-generate the Javadoc template for a method by putting the cursor in the line directly above the method and type `/**` and then `enter`.